projects/canopy/src/lib/brand-icon/brand-icon.component.spec.ts

```diff
@@ -99,20 +99,54 @@ describe('LgBrandIconComponent', () => {
         expect(el.style.fill).toEqual('var(--colour-css-variable)');
       });
     });
+
+    it('when the icon isn\'t coloured it should not set the fill style', () => {
+      when(brandIconRegistryMock.getBrandIcon('sun')).thenReturn(
+        '<svg id="test">test-svg<path id="no-color"></path></svg>',
+      );
+
+      component.name = 'sun';
+
+      fixture.detectChanges();
+      const el = fixture.nativeElement.querySelector('path[id^="lg-brand-icon-"]');
+
+      expect(el.getAttribute('data-colour')).toBeNull();
+      expect(el.style.fill).toEqual('');
+    });
   });
 
-  it('when the icon isn\'t coloured it should not set the fill style', () => {
-    when(brandIconRegistryMock.getBrandIcon('sun')).thenReturn(
-      '<svg id="test">test-svg<path id="no-color"></path></svg>',
-    );
+  describe('the half tone colour input', () => {
+    it('should apply the specific colour', () => {
+      when(brandIconRegistryMock.getBrandIcon('sun')).thenReturn(
+        '<svg id="test">test-svg<path id="Half-tone"></path></svg>',
```

Comment:
Is there a reason to capitalise the id value?

Reply:
It's just how the designers have set them in the svg code. It's a fair chunk of work to change all the layers of all the icons to be lowercase.

Reply:
I assumed that was probably the reason 👍